umm-lib/test/cmr/umm/test/iso_smap/granule.clj

```diff
@@ -116,6 +116,10 @@
    {:orbit
     (umm-g/->Orbit -140.0 1.0 :desc 0.0 :asc)}))
 
+(def generated-granule
```

Comment:
Add a docstring and potentially rename.

Reply:
Added a docstring.  It is a generated granule, so I think the name is okay.